docs/content/en/docs-dev/quickstart/_index.md

```diff
@@ -11,35 +11,41 @@ This page is a guideline for installing PipeCD into your Kubernetes cluster and
 Note: It's not required to install the PipeCD control plane to the cluster where your applications are running. Please read this [blog post](/blog/2021/12/29/pipecd-best-practice-01-operate-your-own-pipecd-cluster/) to understand more about PipeCD in real life use cases.
 
 ### Prerequisites
-- Having a Kubernetes cluster
-- Installed [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [Helm](https://helm.sh/docs/intro/install/) (3.8.0 or later)
+- Having a Kubernetes cluster and connect to it via [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 - Forked the [Examples](https://github.com/pipe-cd/examples) repository
 
-### 1. Installing control plane
+### 1. Installing PipeCD client
+
+The official PipeCD client named `pipectl` can be installed using the following command
 
 ``` console
-helm install pipecd oci://ghcr.io/pipe-cd/chart/pipecd --version {{< blocks/latest_version >}} \
-  --namespace pipecd --create-namespace \
-  --values https://raw.githubusercontent.com/pipe-cd/pipecd/{{< blocks/latest_version >}}/quickstart/control-plane-values.yaml
+OS="darwin" # or "linux"
+curl -Lo ./pipectl https://github.com/pipe-cd/pipecd/releases/download/{{< blocks/latest_version >}}/pipectl_{{< blocks/latest_version >}}_${OS}_amd64
 ```
 
-Once installed, use `kubectl port-forward` to expose the web console on your localhost:
+Then make the pipectl binary executable
 
 ``` console
-kubectl -n pipecd port-forward svc/pipecd 8080
+chmod +x ./pipectl
+```
+
```

Comment:
IMO, I wanna pass through a PATH for pipeclt so, how about moving the binary to pass a PATH?
```
sudo mv ./pipectl /usr/local/bin/pipectl
```